From 14ed1e4ede7ce3f7f391b0d2ca1ebba5826081c6 Mon Sep 17 00:00:00 2001
From: Evan Wang <xswang@marvell.com>
Date: Wed, 11 Jan 2017 00:37:43 +0800
Subject: [PATCH 0801/1345] dts: a3700: add USB2 utmi phy device nodes

commit  bbd931c71726ed8e13a5565ec56f9d1bf6624a9f from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- the utmi phy dts node is disabled as default

Change-Id: I2b3b42732cf404b47bbaf86f0a45bc3969f6f11a
Signed-off-by: Evan Wang <xswang@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/35445
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Victor Gu <xigu@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi |   18 +++++++++++++++++-
 1 file changed, 17 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index 38effb7..8bc9d04 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -383,11 +383,27 @@
 
 			usb2: usb@5e000 {
 				compatible = "marvell,armada-3700-ehci";
-				reg = <0x5e000 0x2000>;
+				reg = <0x5e000 0x1000>;
 				interrupts = <GIC_SPI 17 IRQ_TYPE_LEVEL_HIGH>;
 				status = "disabled";
 			};
 
+			utmi_usb32: utmi@5d000 {
+				compatible = "marvell,armada-3700-utmi-phy";
+				reg = <0x5d000 0x1000>;
+				utmi-port = <UTMI_PHY_TO_USB3_HOST0>;
+				#phy-cells = <0>;
+				status = "disabled";
+			};
+
+			utmi_usb2: utmi@5f000 {
+				compatible = "marvell,armada-3700-utmi-phy";
+				reg = <0x5f000 0x1000>;
+				utmi-port = <UTMI_PHY_TO_USB2_HOST0>;
+				#phy-cells = <0>;
+				status = "disabled";
+			};
+
 			xor@60900 {
 				compatible = "marvell,armada-3700-xor";
 				reg = <0x60900 0x100
-- 
1.7.9.5

